packages/ses/scripts/bundle.js

```diff
@@ -15,10 +17,17 @@ const write = async (target, content) => {
 };
 
 const main = async () => {
-  const bundle = await makeBundle(
+  const bytes = await fs.promises.readFile(
```

Comment:
or `fs.readFileSync` (prob not)

Reply:
No change requested. Here’s some feedback on how I think about this trade-off:

This is a place where it doesn’t matter one way or the other. Synchronous is actually faster. The reason to favor async in Node.js is to reduce variation in latency on a system serving multiple concurrent requests, to mitigate head-of-line blocking in the event loop. Since this is not a server, that’s not a win.

The other good reason to avoid the sync functions is that sync I/O is not portable to the web and embracing async early can avoid an expensive and backward-incompatible refactor in the future if the code needs to be portable to the web later. That is also not a concern here.

The only remaining good reason is to pretend that the sync functions don’t even exist so you don’t have to think about any of the above subtle trade-offs, since the performance win is not worth thinking about.

Reply:
This is my stance too.  I almost never write server code, but I avoid sync methods for the above reasons.  Mostly the "don't have to think" reason, which is truly underrated.